jobs/BAML_LON07506_AVP_Software Developer_ London

```diff
@@ -0,0 +1,54 @@
+---
+title: AVP, Software Developer, Global Technology and Operations
+company: Bank of America Merrill Lynch
+url: http://ig12.i-grasp.com/fe/tpl_bankofamerica06.asp?newms=jj&id=103344&aid=16952
+location: London
+contract: permanent
+contact:
+    name: Janos Meszaros
+    email: janos.meszaros@baml.com
+    ...: ...
+created: !!timestamp '2015-03-24'
+tags:
+  - london
+  - england
+  - uk
+  - python
+  - c++
+  - sql
+---
+
+# This is a header
```

Comment:
You should be able to click the pencil icon here:
https://github.com/pythonjobs/jobs/pull/69/files#
(Right hand side of the page, near the top)

And edit this file online, to change it.  Unfortunately I can't do this as it's 'your' request.